build.sbt

```diff
@@ -162,7 +181,7 @@ lazy val `kyo-core` =
             libraryDependencies += "dev.zio"      %%% "zio-laws-laws"   % "1.0.0-RC27" % Test,
             libraryDependencies += "dev.zio"      %%% "zio-test-sbt"    % "2.1.3"      % Test,
             libraryDependencies += "ch.qos.logback" % "logback-classic" % "1.5.6"      % Test,
-            libraryDependencies += "javassist"      % "javassist"       % "3.12.1.GA"  % Test
+            libraryDependencies += "org.javassist"  % "javassist"       % "3.30.2-GA"  % Test
```

Comment:
I had trouble with `BytecodeTest` failing in `kyo-prelude` and noticed that we're on an old version of `javassist` . Scala steward isn't updating it because the package name changed.